fix for: undefined symbol: _ZN6google8protobuf8internal15ThreadSafeArena13thread_cache_E
see also https://bugs.archlinux.org/task/75242
author: Kenneth Hoste (HPC-UGent)
--- onnx-1.15.0/CMakeLists.txt.orig	2024-01-05 21:11:09.060853188 +0100
+++ onnx-1.15.0/CMakeLists.txt	2024-01-05 21:11:21.587828423 +0100
@@ -46,10 +46,10 @@
 endif()
 
 if(ONNX_USE_PROTOBUF_SHARED_LIBS)
-  if(MSVC)
+	#if(MSVC)
     #TODO: if ONNX_USE_MSVC_STATIC_RUNTIME is ON, it may not work
     add_definitions(-DPROTOBUF_USE_DLLS)
-  endif()
+    #endif()
   set(Protobuf_USE_STATIC_LIBS OFF)
 else()
   set(Protobuf_USE_STATIC_LIBS ON)
